lib/sassc/engine.rb

```diff
@@ -17,6 +17,8 @@ def initialize(template, options = {})
     end
 
     def render
+      return @template if @template.empty?
+
```

Comment:
Perhaps, but it [doesn't preserve the encoding of the input template](https://github.com/sass/sassc-ruby/blob/master/test/engine_test.rb#L165-L170).